packages/mui-base/src/Select/Select.test.tsx

```diff
@@ -660,11 +660,32 @@ describe('<Select />', () => {
       });
 
       expect(handleChange.callCount).to.equal(1);
-      expect(handleChange.args[0][0]).to.haveOwnProperty('type', 'click');
```

Comment:
for some reason this test is failing, so converted this test to `expect(handleChange.args[0][0].type).to.equal('click');`. I belive both are testing same thing